websites/A/Apple/metadata.json

```
@@ -8,6 +8,10 @@
 		{
 			"name": "Drull",
 			"id": "244953147154366464"
+		},
+		{
+			"name": "Yan.",
+			"id": "735538297815957584"
```

```suggestion
		}
```

You didn't edit anything of the codebase, so please remove this

https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md
> If you make modifications to a presence and change at **least a quarter** of the presence's **codebase**, you are allowed to add yourself as a contributor. Contact a reviewer for more information about this subject.